controllers/posts_controller.js

```diff
@@ -1,66 +1,60 @@
-const post = require('../model/posts_model')
-const utils = require('../utils/util')
+const post = require('../model/posts_model');
+const utils = require('../utils/util');
 
-module.exports.addPosts = ({body: {url = null, description = null, tags = null}, user = null}, res) => {
-    let errorMessage = '';
+module.exports.addPosts = async ({body: {url = null, description = null, tags = null}, user = null}, res) => {
+    let errors = [];
 
     if (utils.isEmpty(user.id)) {
-        errorMessage += 'User ID is Null\n'
+        errors.push('User ID is Null');
     }
     if (utils.isEmpty(url)) {
-        errorMessage += 'URL is Null\n'
+        errors.push('URL is Null');
     }
 
-    if (!utils.isEmpty(errorMessage)) {
-        res.status(401).json({
+    if (errors.length > 0) {
+        res.status(400).json({
             error: true,
-            data: errorMessage
+            data: errors.join('\n')
         })
```

Comment:
I'd `return;` after the `res.status.json()` so you don't have to start a `else` block after; that way you don't have to have a whole new block (indentation hell)

Even better would be to simply `return res.json()...` so it's done at the same time

Reply:
@silverAndroid

Reply:
Yeah we can do that, this way I don't have to worry about ESLint complaining about no returns with arrow functions 😂

Reply:
I'm already doing it in users controller